R/print.data.table.R

```diff
@@ -247,7 +247,7 @@ char.trunc = function(x, trunc.char = getOption("datatable.prettyprint.char")) {
   nchar_width = nchar(x, 'width') # Check whether string is full-width or half-width, #5096
   nchar_chars = nchar(x, 'char')
   is_full_width = nchar_width > nchar_chars
-  idx = pmin(nchar_width, nchar_chars) > trunc.char
+  idx = pmin(nchar_width, nchar_chars) > trunc.char & !is.na(x)
```

Comment:
I _guess_ it will be better to handle this with the keepNA argument to nchar() instead... I can take a look a bit later

Reply:
NVM, I think we need `idx` to be "complete" (no `NA`), so this works.

Add NEWS?

Reply:
Sure! Does this fall under bug or note?

Reply:
this is a bug fix -- normal usage of the package produced an error, that's a bug